TheFourFifths/consus
# [TFF-220] Equipment Saving

### Summary

We can now save equipment to be used later. This PR does not include the ability to retrieve/unsave equipment.

### Checklist


### How to Review

1. Run both the server and client on `item-save-TFF-220`.
2. Create a model that can be checked out.
3. Check out a model and an item.
4. Save them both.

### Links

- [TFF-220](https://msoese.atlassian.net/browse/TFF-220)
- TheFourFifths/consus-client/pull/111
